src/components/Main/Main.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import Nav from '../Nav/Nav.js';
+import NotFound from '../NotFound/NotFound.js';
+import RGB from '../RGB/RGB.js';
+import './Main.css';
+
+export default function Main() {
+  return (
+    <main>
+      <Nav />
+      <Switch>
+        <Route path="/rgb/:r/:g/:b" component={RGB} />

```

Comment:
This is working, but there is no path when the page loads. For example, "/" will always be the first path that loads; without giving that path a component or redirecting, your app always starts with a "404 not found" in its current state. If you would like some advice on handling this, let me know!

Reply:
I think I just forgot to add this too.. I'll add this to the things I need to take care of during hack-day tomorrow.